### DISCUSSION TITLE
Viewport resizing to window size

### DISCUSSION THREAD

**Discussion by Shivam93937** [GoogleChrome/lighthouse#15956] [General]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://google.com

### What happened?

When running from CLI, is there an option to by default set the viewport according to the window size because it resizes to cover the window size after certain amount of time.

### What did you expect?

I expected that the viewport should automatically be sized to window size.

### What have you tried?

I have tried changing the puppeteer launch options,  via lighthouse configurations, and screenemulation options.

### How were you running Lighthouse?

CLI

### Lighthouse Version

11.7.1

### Node Version

20.11.0

### OS

Windows

**Comment by adamraine** [comment]
The CLI flag `--screenEmulation.disabled` should prevent Lighthouse from changing the emulated screen size. Also available on the config:

```js
const config = {
  extends: 'lighthouse:default',
  settings: {
    screenEmulation: {disabled: true},
  },
}
```

Are you using Puppeteer as well?

**Reply by Shivam93937** [reply]
Yes I am using puppeteer as well

**Reply by adamraine** [reply]
`puppeteer.connect` applies it's own emulated screen size by default. Try using `defaultViewport: null` on the connect options `puppeteer.connect({browserURL: 'http://localhost:9222', defaultViewport: null})`

**Comment by Shivam93937** [comment]
Do you have any idea how to emulate mobile screen without resizing viewport to window size?
